commands/cluster_commands/install

```diff
@@ -97,11 +97,19 @@ if [ "${TARGET_PLATFORM}" = $CIVO ]; then
   # first check if civo is installed or not, if not? then install it
   check_and_install_civo
 
+  if grep '"apikeys":{}' ~/.civo.json; then
+    CIVO_API_KEY=$(prompt_for_input_with_validation "Enter CIVO API KEY: " "Provide the civo api key to be used - valide api key is required" "$non_empty_regex" "Invalid api key format. Please try again." || exit $?)
+    civo apikey add grapple $CIVO_API_KEY
+    civo apikey current grapple
+  fi
+
   CIVO_REGION=$(if [ "${CIVO_REGION}" != "" ]; then echo ${CIVO_REGION}; else gum spin --title "fetching civo regions" --show-output -- civo region ls -o custom -f code | gum choose; fi)
   civo region use ${CIVO_REGION} | true
 
   CIVO_CLUSTER=$(if [ "${CIVO_CLUSTER}" != "" ]; then echo ${CIVO_CLUSTER}; else if gum spin --title "fetching civo clusters" -- civo k8s ls -o custom -f name; then civo k8s ls -o custom -f name | gum choose; else echo ""; fi; fi)
   echo "selected civo cluster: ${CIVO_CLUSTER}"
+  current_user=$(whoami)
+  sudo chown -R "${current_user}":"${current_user}" /home/"${current_user}"/.kube
```

Comment:
@zahid-iqbal-alpha 
sudo brings up a password request and the chown command doesn't work for me:
<img width="623" alt="image" src="https://github.com/grapple-solutions/grapple-cli/assets/130764623/e35fa6e7-b248-45f1-8f05-9e95b85e8141">

current user is not always equals to the group of the user.
this command would return the primary group name:
```
id -gn $(whoami)
```

Reply:
@zahid-iqbal-alpha 
please do not use sudo here...
what is it, that you are trying to achieve exactly?

${HOME}/.kube
should usually always be owned by the current user...

Reply:
@grapple-solutions when I am using 
docker run --rm -v ${HOME}/.kube:/home/linuxbrew/.kube/ -it homebrew/brew bash

Reply:
This was happening on linux but Does this happens on mac as well?

Reply:
But I think that, this problem won't occur in real env.

Reply:
@zahid-iqbal-alpha 
With the latest changes you fixed the behaviour of kubectl. There is no need for the volume mount anymore.
so you can just omit the -v from the command:
from:
`docker run --rm -v ${HOME}/.kube:/home/linuxbrew/.kube/ -it homebrew/brew bash`
to:
`docker run --rm -it homebrew/brew bash`
then the issue won't happen anymore.
and it shouldn't happen in any other real customers machine, because the .kube directory should always already have the right ownership (our case with docker was a bit special / a workaround)